README.md

```diff
@@ -111,7 +111,7 @@ Permissions that applications may use (names are subject to change):
 | Location | Use GPS and positioning. |
 | MediaIndexing | Access to Tracker to list files on device. If you have access to a data directory, you may want to use also this. |
 | Messages | Access to message data and to send SMS messages. |
-| WebView | If you use Gecko based WebView you need this. |
+| Microphone | Record audio with microphone. Gives access to audio devices for playback as well. |
```

Comment:
Here I would just mention that "Record audio with microphone." and perhaps recommend Audio for playback.

Reply:
Added with 36f9ad3